emails/utils.py

```diff
@@ -53,36 +60,10 @@ def get_email_domain_from_settings():
     return email_network_locality
 
 
-def get_domains_from_settings():
-    return {
-        'RELAY_FIREFOX_DOMAIN': settings.RELAY_FIREFOX_DOMAIN,
-        'MOZMAIL_DOMAIN': settings.MOZMAIL_DOMAIN
-    }
-
-
-@time_if_enabled('ses_send_email')
-def ses_send_email(from_address, to_address, subject, message_body):
-    emails_config = apps.get_app_config('emails')
-    try:
-        emails_config.ses_client.send_email(
-            Destination={'ToAddresses': [to_address]},
-            Message={
-                'Body': message_body,
-                'Subject': {'Charset': 'UTF-8', 'Data': subject},
-            },
-            Source=from_address,
-            ConfigurationSetName=settings.AWS_SES_CONFIGSET,
-        )
-        incr_if_enabled('ses_send_email', 1)
-    except ClientError as e:
-        logger.error('ses_client_error', extra=e.response['Error'])
-        return HttpResponse("SES client error", status=400)
-    return HttpResponse("Sent email to final recipient.", status=200)
-
-
-@time_if_enabled('ses_send_email')
+@time_if_enabled('ses_send_raw_email')
 def ses_send_raw_email(
-        from_address, to_address, subject, message_body, attachments
+    from_address, to_address, subject, message_body, attachments,
+    reply_address, mail, address
```

Comment:
I'm sure there's some more clean-up that can be done here now that this function has the entire `mail` object in it.

Reply:
[Not Required] Some code we can refactor into a separate function:
- attachment
- reply object creation (this can after where the `ses_send_raw_email` is called)

Reply:
good call ... much nicer function break-down now in https://github.com/mozilla/fx-private-relay/pull/1038/commits/901310c81f8b2a5d1eb2054d28ffa44108a5a83e ... new functions: 
* _start_message_with_headers
* _add_body_to_message
* _add_attachments_to_message
* _store_reply_record